cypress/e2e/Batch_Canceled.js

```diff
@@ -45,6 +45,8 @@ describe("Batch is canceled with player in intro steps", () => {
         cy.wait(3000)
         cy.contains("About").should("not.exist");
         // TODO @kailyl: Check that sorry page displays here
+        cy.contains("Your experiment unexpectedly stopped. You have received $0.00 for participating in our experiment.")
```

Comment:
The player should still be paid for their time here. We need to update this test to compute that.